geode-core/src/test/java/org/apache/geode/test/dunit/rules/DistributedRestoreSystemProperties.java

```diff
@@ -30,7 +27,10 @@
 public class DistributedRestoreSystemProperties extends RestoreSystemProperties
```

Comment:
I don't know why you are calling the the before method of RestoreSystemProperties reflectively, but since we are doing that, do you still need to have DistributedRestoreSystemProperties extend RestoreSystemProperties anymore?

Reply:
DistributedRestoreSystemProperties isa RestoreSystemProperties and some tests use both:

```java
public RestoreSystemProperties restoreSystemProperties = new DistributedRestoreSystemProperties();
```

Normally the subclass could invoke protected methods on the superclass. However, Java won't allow this when delegating to multiple static instances of RestoreSystemProperties even from a subclass. I originally tried to code it the way you suggest.